src/buildTree.js

```diff
@@ -12,13 +12,14 @@ const INHERITABLE_STYLES = [
   'fontSize',
   'fontStyle',
   'fontWeight',
+  'textAlign',
+  'textDecoration',
```

Comment:
Hi Jon, I actually wanted to use the exact same API of textDecorationLine but I found this:
https://github.com/airbnb/react-sketchapp/blob/master/src/stylesheet/expandStyle.js#L61

When the style gets passed down as a textStyle this part expands it to be renamed `textDecoration`; I don't know why this is needed. I can go ahead and remove the shortHand here and add the `textDecorationLine` over here so it's propagated like it. Thoughts?